RCAIDE/Library/Methods/Aerodynamics/Common/Lift/BET_calculations.py

```diff
@@ -118,12 +124,14 @@ def compute_airfoil_aerodynamics(beta,c,r,R,B,Wa,Wt,a,nu,airfoils,a_loc,ctrl_pts
         #This is an atrocious fit of DAE51 data at RE=50k for Cd
         Cdval = (0.108*(Cl*Cl*Cl*Cl)-0.2612*(Cl*Cl*Cl)+0.181*(Cl*Cl)-0.0139*Cl+0.0278)*((50000./Re)**0.2)
         Cdval[alpha>=np.pi/2] = 2.
-
+        
+        alpha_disc = np.tile(alpha[:,:, None], (1, 1, Nr)) 
+        Re_disc    = np.tile(Re[:,:, None], (1, 1, Nr))  
 
     # prevent zero Cl to keep Cd/Cl from breaking in BET
     Cl[Cl==0] = 1e-6
 
-    return Cl, Cdval, alpha, Ma, W, Re  
+    return Cl, Cdval, alpha, alpha_disc,Ma,W,Re,Re_disc
```

Comment:
returns disk Re and alpha for rotor calculations

Reply:
This change was made so that we don't have to do the computation in the noise script